test/image/mocks/bar-rounded-corners.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Let's rename this new mock to `zz-bar-rounded-corners.json` to avoid hitting image test failures.
Thank you!

Reply:
Will do!